winterfell/webapp/src/templates/modals/confirmClaimModifyContact.jade

```diff
@@ -0,0 +1,26 @@
+// For Confirm Claim page
+#vfi-confirm-modify-contact-modal
+  button.close-modal-btn(ng-click="onCloseModal()")
+    div.close-icon
+  h3 {{modalTitle}}
+
+  div.input-wrapper(ng-if="forType == 'address'")
+    input.name(placeholder="Location Name i.e. Home, Work")
+    input.sender(placeholder="Send this to...")
+    input.street1(placeholder="Street Number & Name")
+    input.street2(placeholder="i.e. Suite, Apartment, Room, P.O., etc.")
+    input.city(placeholder="City")
+    input.country(placeholder="Country")
+    input.province(placeholder="State/Province")
+    input.zip(placeholder="Zip/Postal")
+
+  div.input-wrapper(ng-if="forType == 'email'")
+    input(placeholder="Contact Name")
+    input(placeholder="Email")
+    input(placeholder="Confirm Email")
+
+  div.button-wrapper
+    button.cancel-btn(ng-click="onCloseModal()", ng-if="targetIndex < 0") Cancel
+    button.delete-btn(ng-click="onDelete()", ng-if="targetIndex >= 0") Remove
+    button.save-btn(ng-click="onSave()") Save
```

Comment:
Could do something like `ng-click=close({$value: address})` and catch the return value from uibModal.open and then save that return value to the model in the parent controller.